otlp/measure.go

```diff
@@ -0,0 +1,122 @@
+package otlp
+
+import (
+	"fmt"
+
+	"github.com/segmentio/stats/v4"
+	commonpb "go.opentelemetry.io/proto/otlp/common/v1"
+	metricpb "go.opentelemetry.io/proto/otlp/metrics/v1"
+)
+
+func convertMetrics(metrics ...metric) []*metricpb.Metric {
+	mm := []*metricpb.Metric{}
+
+	for _, metric := range metrics {
+		attributes := tagsToAttributes(metric.tags...)
+
+		m := &metricpb.Metric{
+			Name: fmt.Sprintf("%s.%s", metric.measureName, metric.fieldName),
```

Comment:
If it's just strings i'd just do `metric.measureName + "." + metric.FieldName` and avoid the Sprint parse overhead.